stratum/hal/lib/bcm/bcm_switch.cc

```diff
@@ -401,6 +402,18 @@ ::util::Status BcmSwitch::RetrieveValue(uint64 /*node_id*/,
         resp.mutable_node_packetio_debug_info()->set_debug_string(
             "A (sample) node debug string.");
         break;
+
+      case DataRequest::Request::kOpticalChannelInfo: {
+        ::util::Status status = phal_interface_->GetOpticalTransceiverInfo(
+            req.optical_channel_info().node_id(),
+            req.optical_channel_info().port_id(),
+            resp_val.mutable_optical_channel_info());
+        if (status.ok()) {
+          resp = resp_val;
+        }
```

Comment:
can be:
```
        status.update(phal_interface_->GetOpticalTransceiverInfo(
            req.optical_channel_info().node_id(),
            req.optical_channel_info().port_id(),
            resp.mutable_optical_channel_info()));
```

no need to have a new variable `resp_val`

Reply:
Fixed. Please review.